### DISCUSSION TITLE
Bundle Mode Pack Separately support?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [jelte/be.khepri.play.assetdelivery.addressables#19] [Ideas]
I want to use a Bundle Mode with Pack Separately option. As a workaround for now I use a bunch of bundles.
Just wanted to ask if you are planing to support "Pack Separately" option?
Maybe you can guide me a little bit, and I can implement it myself...

**Comment by jelte** [comment]
definitely something I would like to have a look at.  There is a major problem with this, which is that Google only supports 150 packs. 

For this the assetPacks would need to be build differently as currently it's one-to-one.
Added https://github.com/jelte/be.khepri.play.assetdelivery.addressables/issues/22, and will look into it on the weekend.
